src/components/UI/dropdown/index.js

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState, useRef } from 'react';
+import Link from 'next/link';
+import styles from './dropdown.module.scss';
+
+const Dropdown = ({
```

Comment:
where are the PropTypes defined for this component?

Reply:
I guess because This is JavaScript file not a TypeScript file,I  don't know that we have to define PropTypes here also, please guide me.